virttest/qemu_devices/qdevices.py

```diff
@@ -1631,14 +1632,24 @@ def _convert_memobj_args(args):
         :return: Converted args.
         :rtype: dict
         """
+        command_dict = {}
         for key, val in args.items():
-            if "size" == key:
-                args[key] = int(utils_numeric.normalize_data_size(val, "B"))
-            # "share" from object( "qom-type": "memory-backend-memfd" )
-            # "reserve" from object( "qom-type": "memory-backend-memfd" )
-            # "hugetlb" from object( "qom-type": "memory-backend-memfd" )
-            elif key in ("share", "reserve", "hugetlb"):
-                args[key] = val == "yes"
+            if key in ("size", "align"):
+                command_dict[key] = int(utils_numeric.normalize_data_size(val,
+                                                                          "B"))
+            # "share", "reserve", "hugetlb"
+            # from object( "qom-type": "memory-backend-memfd" )
+            # "prealloc", "dump", "merge"
+            # from -object ("qom-type": "memory-backend-ram")
+            # readonly from -object("qom-type": "memory-backend-file")
+            elif key in ("share", "reserve", "hugetlb",
+                         "prealloc", "dump", "merge", "readonly"):
+                command_dict[key] = val.lower() in ("yes", "on")
```

Comment:
> I am confused about the value here, I think that means the user could set the corresponding values starting with the capital letter if we can handle the situation `val.lower() in ("yes", "on")` Like cfg:
> 
> ```
> share = Yes
> # share = YES
> readonly = YeS
> # readonly = YES
> ```
> 
> But usually, we did not suggest using the above format for the value in order to keep the consistency in the current usage of the framework. Since we used to define the format, the value is lowercase for the switch type parameters interface. Refer to base.cfg
> 
> Hi @nickzhq Is there a special consideration from your side? Thanks. CC @zhencliu

Hi @YongxueHong , while I debugging the codes before, the values you mentioned may be one of Yes, yes, On, on( they may come from qemu-code or user-side ). Therefore, the val.lower() is a generic approach to handle this issue.

Thanks! 
cc @zhencliu